lib/ansible/plugins/callback/logdna.py

```diff
@@ -0,0 +1,147 @@
+# (C) 2018, Samir Musali <samir.musali@logdna.com>
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: logdna
+    callback_type: aggregate
+    short_description: Sends playbook logs to LogDNA
+    description:
+      - This callback will report logs from playbook actions, tasks, and events to LogDNA (https://app.logdna.com)
+    version_added: 0.1
+    requirements:
+      - whitelisting in configuration
+    options:
+      conf_key:
+        required: True
+        description: LogDNA Ingestion Key
+        env:
+          - name: LOGDNA_INGESTION_KEY
```

Comment:
not a blocker: aside from environment variables, you can also use ini entries in ansible.cfg, it is a nice option for users to be able to configure in the way they prefer.